doc/md/CLI/welcome.md

```diff
@@ -0,0 +1,40 @@
+---
+id: Welcome to Atlas
+slug: /CLI/
+sidebar_position: 2
+---
+
+## CLI
+
+Work with any data source from the command line.
+
+![Alt Text](https://release.ariga.io/images/assets/atlas-intro.gif)
+
+## Installation
+
+Download from one of our links or using curl.
+```shell
+curl -LO https://release.ariga.io/atlas/atlas-{os}-{arch}-{version}
```

Comment:
give a copy-able instructions for each platform. 
reduce friction for the user.